sonar-project.properties

```diff
@@ -5,3 +5,6 @@ sonar.organization=broad-databiosphere
 sonar.sources=src
 # Skip test files, which can be anywhere in the source tree.
 sonar.exclusions=**/**.test.js
+
+sonar.tests=src
+sonar.test.inclusions=**/**.test.js
```

Comment:
Interesting. I looked at these settings, but I didn't see how they would change the sonar report. Is the idea that there are different checks run on test code vs non-test code?

Also It looks like the coverage data is still including `.test.js` files. https://sonarcloud.io/component_measures?id=DataBiosphere_terra-ui&metric=coverage&view=list

Maybe try adding 
```
sonar.coverage.exclusions=**/**.test.js
```
As well?

And now that I take a closer look, I see that we have both `.test.js` and `.test.ts`. So we may need to include/exclude both patterns via regex.

Reply:
Yeah, I just arrived at the same conclusion. Your PR (#3628) fixed the missing coverage and it looks like the test files showing up in the coverage report are `.test.ts` files (which makes sense since we're only ignoring `.test.js`).

I don't know what effect the test scope configuration has on SonarCloud, but based on https://docs.sonarqube.org/latest/project-administration/narrowing-the-focus/, it seems like it might be worth configuring this?